src/components/AffectedOfferings.vue

```diff
@@ -50,6 +50,10 @@ const isAnythingExpanded = computed(() => (
   || Object.values(expandedModules.value).some(Boolean)
 ));
 
+const areAllModulesCollapsed = computed(() => (
+  !Object.values(expandedModules.value).every(Boolean)
+));
```

Comment:
```suggestion
const isAnythingCollapsed = computed(() => (
  !Object.values(expandedModules.value).some(Boolean)
));
```

Reply:
The actual code returns true if any module is collapsed (notice the negation at the beginning of the statement). So I believe we could just correct the name to `isAnythingCollapsed` as you suggested.

I think your suggestion to change `every` to `some` would make the statement become false and hiding the button in the moment any module is expanded.

WDYT?

Reply:
Oops, you're right, I got confused with the name and got carried away by the `isAnythingExpanded` implementation...

Reply:
Yea the name is totally wrong and very confusing, so good job rising this!
I'll amend it soon.